datasette/views/table.py

```diff
@@ -543,29 +549,35 @@ async def data(
         sortable_columns = await self.sortable_columns_for_table(
             database, table, use_rowid
         )
-
-        # Allow for custom sort order
-        sort = special_args.get("_sort")
-        sort_desc = special_args.get("_sort_desc")
-
-        if not sort and not sort_desc:
-            sort = table_metadata.get("sort")
-            sort_desc = table_metadata.get("sort_desc")
-
-        if sort and sort_desc:
-            raise DatasetteError("Cannot use _sort and _sort_desc at the same time")
-
-        if sort:
-            if sort not in sortable_columns:
-                raise DatasetteError(f"Cannot sort table by {sort}")
-
-            order_by = escape_sqlite(sort)
-
-        if sort_desc:
-            if sort_desc not in sortable_columns:
-                raise DatasetteError(f"Cannot sort table by {sort_desc}")
-
-            order_by = f"{escape_sqlite(sort_desc)} desc"
+        #
+        # Getting sorting conditions from url and saving them in a list
+        #
+        sort = [
+            SortingOrder(escape_sqlite(value), "asc" if name == "_sort" else "desc")
+            for name, value in query_params
+            if name == "_sort" or name == "_sort_desc"
+        ]
+        sort_asc = []
+        sort_desc = []
+        if not sort:
+            if table_metadata.get("sort"):
+                sort.append(SortingOrder(table_metadata.get("sort"), "asc"))
+            if table_metadata.get("sort_desc"):
+                sort.append(SortingOrder(table_metadata.get("sort_desc"), "desc"))
+        for item in sort:
```

Comment:
nit: maybe you could create `sort_asc` and `sort_desc` using list comprehension.